tests/integration/framework/util/rand.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"crypto/rand"
+)
+
```

Comment:
```suggestion
func init() {
    rand.Seed(time.Now().UnixNano())
}
```

Reply:
`crypto/rand` is a CSPRNG that uses entropy from the kernel, so `rand.Seed`  isn't needed (it should not even work)